src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random number = new Random();
```

Comment:
naming is important, this is not a number, but a class that can provide us with all sorts of random values. Just `random` would be fine